server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserQuotaRoutes.java

```diff
@@ -139,6 +151,88 @@ public void defineGetQuota() {
         }, jsonTransformer);
     }
 
+    @GET
+    @ApiOperation(
+        value = "Reading count and size at the same time",
+        notes = "If there is no limitation for count and/or size, the returned value will be -1"
+    )
+    @ApiImplicitParams({
+        @ApiImplicitParam(
+                required = false,
+                name = "minOccuptationRatio",
+                paramType = "query parameter",
+                dataType = "Double",
```

Comment:
why do we want double precision instead of float ? looks overkill to me

Reply:
Why not?

Reply:
Ok, then why not BigDecimal ?

Reply:
Because we use a **double** in our java code.

Why should we introduce some precision mismatch? If **double** is not the right precision at the API level, why do we keep it at the java level?

Reply:
That's right, I missed that in the first place. Still I think it doesn't make sense (when using double, you express the intent of being very precise, that's clearly not the case).
I won't fight for this kind of details, but types carry more intent then what one would think, even low level types.